plugins/formal-verification/tests/org/jetbrains/kotlin/formver/plugin/services/ExtensionRegistrarConfigurator.kt

```diff
@@ -9,35 +9,32 @@ import org.jetbrains.kotlin.compiler.plugin.CompilerPluginRegistrar.ExtensionSto
 import org.jetbrains.kotlin.config.CompilerConfiguration
 import org.jetbrains.kotlin.fir.extensions.FirExtensionRegistrarAdapter
 import org.jetbrains.kotlin.formver.*
-import org.jetbrains.kotlin.test.model.TestFile
+import org.jetbrains.kotlin.formver.plugin.services.FormVerDirectives.ALWAYS_VALIDATE
+import org.jetbrains.kotlin.formver.plugin.services.FormVerDirectives.FULL_VIPER_DUMP
+import org.jetbrains.kotlin.formver.plugin.services.FormVerDirectives.NEVER_VALIDATE
+import org.jetbrains.kotlin.formver.plugin.services.FormVerDirectives.RENDER_PREDICATES
+import org.jetbrains.kotlin.test.directives.model.DirectivesContainer
+import org.jetbrains.kotlin.test.directives.model.SimpleDirectivesContainer
 import org.jetbrains.kotlin.test.model.TestModule
 import org.jetbrains.kotlin.test.services.EnvironmentConfigurator
 import org.jetbrains.kotlin.test.services.TestServices
 
 class ExtensionRegistrarConfigurator(testServices: TestServices) : EnvironmentConfigurator(testServices) {
-    private fun <V> List<TestFile>.retrieveByPathMatch(map: Map<String, V>, default: V): V {
-        for ((k, v) in map) {
-            if (any { it.originalFile.absolutePath.contains(k) }) {
-                return v
-            }
-        }
-        return default
-    }
+    override val directiveContainers: List<DirectivesContainer>
+        get() = listOf(FormVerDirectives)
 
     override fun ExtensionStorage.registerCompilerExtensions(module: TestModule, configuration: CompilerConfiguration) {
-        val logLevel = module.files.retrieveByPathMatch(
-            mapOf(
-                "full_viper_dump" to LogLevel.FULL_VIPER_DUMP,
-                "predicates" to LogLevel.SHORT_VIPER_DUMP_WITH_PREDICATES
-            ), default = LogLevel.SHORT_VIPER_DUMP
-        )
+        val logLevel = when {
+            FULL_VIPER_DUMP in module.directives -> LogLevel.FULL_VIPER_DUMP
+            RENDER_PREDICATES in module.directives -> LogLevel.SHORT_VIPER_DUMP_WITH_PREDICATES
+            else -> LogLevel.SHORT_VIPER_DUMP
+        }
```

Comment:
It may be worth checking here that someone doesn't accidentally use both `FULL_VIPER_DUMP` and `RENDER_PREDICATES`.